src/Simplify.elm

```diff
@@ -1575,13 +1538,22 @@ type alias CheckInfo =
     , inferredConstants : ( Infer.Inferred, List Infer.Inferred )
     , parentRange : Range
     , fnRange : Range
-    , firstArg : Node Expression
-    , secondArg : Maybe (Node Expression)
-    , thirdArg : Maybe (Node Expression)
     , usingRightPizza : Bool
+    , firstArg : Node Expression
+    , argsAfterFirst : Array (Node Expression)
```

Comment:
Not that it matters much- Should we represent the arguments as an `Array` or a `List`?